src/node_buffer.cc

```diff
@@ -492,9 +492,8 @@ void Copy(const FunctionCallbackInfo<Value> &args) {
 
   THROW_AND_RETURN_UNLESS_BUFFER(env, args[0]);
   THROW_AND_RETURN_UNLESS_BUFFER(env, args[1]);
-  Local<Object> buffer_obj = args[0].As<Object>();
+  ArrayBufferViewContents<char> source(args[0]);
   Local<Object> target_obj = args[1].As<Object>();
-  SPREAD_BUFFER_ARG(buffer_obj, ts_obj);
   SPREAD_BUFFER_ARG(target_obj, target);
```

Comment:
Suggestion: un-const `data()` (or add a `data_mut()` method) so you can ArrayBufferViewContents-ify `target` as well.

Reply:
@bnoordhuis Unfortunately, it doesn’t work quite that easily for writing – V8 does currently not provide a way to write into on-heap typed arrays (without materializing their `ArrayBuffer`s). I’m looking into addressing that, though.